kotest-assertions/kotest-assertions-core/api/kotest-assertions-core.api

```diff
@@ -1862,6 +1871,8 @@ public final class io/kotest/matchers/ranges/Range {
 	public fun <init> (Lio/kotest/matchers/ranges/RangeEdge;Lio/kotest/matchers/ranges/RangeEdge;)V
 	public final fun component1 ()Lio/kotest/matchers/ranges/RangeEdge;
 	public final fun component2 ()Lio/kotest/matchers/ranges/RangeEdge;
+	public final fun contains (Lio/kotest/matchers/ranges/Range;)Z
+	public final fun contains (Lio/kotest/matchers/ranges/RangeEdge;)Z
```

Comment:
Why is this entire class not internal?

Reply:
> Why is this entire class not internal?

it's internal now, TY for calling it out